crates/egui/src/containers/collapsing_header.rs

```diff
@@ -66,6 +70,14 @@ impl CollapsingState {
         self.state.open = open;
     }
 
+    pub fn is_collapsed(&self) -> bool {
```

Comment:
Easier to read/write with when the `Window` contained usages of both the word 'open' and 'collapse'